packages/react/tooltip/src/Tooltip.tsx

```diff
@@ -235,7 +235,7 @@ const TooltipTrigger = React.forwardRef<TooltipTriggerElement, TooltipTriggerPro
           data-state={context.stateAttribute}
           {...triggerProps}
           ref={composedTriggerRef}
-          onMouseEnter={composeEventHandlers(props.onMouseEnter, context.onOpen)}
+          onMouseOver={composeEventHandlers(props.onMouseOver, context.onOpen)}
```

Comment:
There is an issue with this... If a dialog close button (for example) is positioned on top of a tooltip trigger, when the dialog closes the tooltip will open:

https://user-images.githubusercontent.com/175330/145034878-49fa44d9-3076-4932-9f92-ee51eebac977.mp4

The alternative is to try adding `pointer-events: none` to the content parts but `Popover` has a wrapping div that cannot be styled.

Reply:
I'm not sure it's an issue actually. To be honest, I kinda like it that it does that!

Reply:
Furry muff 🎉